examples/contrib/readhar.py

```diff
@@ -0,0 +1,134 @@
+"""Reads HAR files into flow objects"""
+import asyncio
+import codecs
+import json
+import logging
+import time
+
+from mitmproxy import command
+from mitmproxy import connection
+from mitmproxy import ctx
+from mitmproxy import exceptions
+from mitmproxy import http
+from mitmproxy import types
+
+
+class ReadHar:
+    def __init__(self):
+        self.harFile = {}
+        self.flows = []
+
+    def fix_headers(self, request_headers):
+        """Changes provided headers into flow-friendly format"""
```

Comment:
Nit: This method does not _change_ the passed headers, it converts/transforms/... the passed headers. The passed value remains unchanged.

"flow-friendly format" is a bit nondescriptive. Let's state right away that it converts into a list of `(b"header-name", b"header-value")` tuples?